pkg/ingestor/parser/slsa/parser_slsa.go

```diff
@@ -106,39 +113,42 @@ func (s *slsaParser) getSubject() error {
 }
 
 func (s *slsaParser) getMaterials() error {
-	switch s.header.PredicateType {
+	switch s.smt.PredicateType {
 	case slsa01.PredicateSLSAProvenance:
-		if err := s.getMaterials0(s.s01smt.Predicate.Materials); err != nil {
+		if err := s.getMaterials0(s.pred01.Materials); err != nil {
 			return err
 		}
 	case slsa02.PredicateSLSAProvenance:
-		if err := s.getMaterials0(s.s02smt.Predicate.Materials); err != nil {
+		if err := s.getMaterials0(s.pred02.Materials); err != nil {
 			return err
 		}
-	case slsa1.PredicateSLSAProvenance:
-		if err := s.getMaterials1(s.s1smt.Predicate.BuildDefinition.ResolvedDependencies); err != nil {
+	case smtslsa1.PredicateSLSAProvenance:
```

Comment:
should this not also use the new in-toto attestation library? Is still pointing to in-toto-golang?

Reply:
meaning it should be `slsa1`?

Reply:
It is used being for the checking the version which is defined [here](https://github.com/in-toto/in-toto-golang/blob/72262c0fd3a9c392440040c04336174eddac427b/in_toto/slsa_provenance/v1/provenance.go#L11).